dev-src/dev/db.clj

```diff
@@ -18,8 +18,11 @@
   (db/transact seed-tx-data))
 
 (defn reset-db! []
-  (d/delete-database (config :datomic :uri))
-  (db/initialize)
+  ;; TODO: This isn't really correct. In Datomic you shouldn't delete and
```

Comment:
Hmm... It attempts to explain how it's wrong. Also it's in `dev-src` code, so nothing that's run in production.

Doesn't adding this comment improve things over not having the comment at all? The problem was always there.

What more would you like to see added?